backend/db_interface.py

```diff
@@ -0,0 +1,52 @@
+from pymongo import MongoClient, errors
+
+VERWIKI_DB_NAME = 'verwiki'
+TREES_TABLE_NAME = "radialTrees"
```

Comment:
lets be consistent and use either single quote or double quote strings throughout.

Reply:
Can we make this part of the linting step of the pipeline? Otherwise it's a losing battle

Reply:
Nevermind, turns out that this step is already part of the `black` linter